modules/cert-manager-install-cli.adoc

```diff
@@ -0,0 +1,129 @@
+// Module included in the following assemblies:
+//
+// * security/cert_manager_operator/cert-manager-operator-install.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cert-manager-install-cli_{context}"]
+= Installing the {cert-manager-operator} by using the CLI
+
+.Prerequisites
+
+* You have access to the cluster with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a new project named `cert-manager-operator` by running the following command:
++
+[source, terminal]
+----
+oc new-project cert-manager-operator
+----
+
+. Create an `OperatorGroup` object:
+
+.. Create a YAML file, for example, `operatorGroup.yaml`, with the following content:
++
+[source, yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-cert-manager-operator
+  namespace: cert-manager-operator
+spec:
+  targetNamespaces:
+  - "cert-manager-operator"
+----
+
+.. Create the `OperatorGroup` object by running the following command:
++
+[source, terminal]
+----
+oc create -f operatorGroup.yaml
+----
+
+. Create a `Subscription` object:
+
+.. Create a YAML file, for example, `subscription.yaml`, that defines the `Subscription` object:
++
+[source, yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-cert-manager-operator
+  namespace: cert-manager-operator
+spec:
+  channel: stable-v1
+  name: openshift-cert-manager-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  installPlanApproval: Automatic
+  startingCSV: cert-manager-operator.v1.13.0
```

Comment:
If you cherry-pick/backport this PR on an older OCP branch, please ensure to use respective cert-manager version here in the `startingCSV` field.

Reply:
That's a great point!  I am not planning to push the changes to 4.12 since it will be most likely EOL in another 3 months and the changes do not comprise of any bug fixes. So I think I can keep this as is.